.travis.yaml

```diff
@@ -0,0 +1,18 @@
+language: python
+python:
+  - "3.6"
+
+# command to install dependencies
+install: "pip install -r requirements.txt"
+
+# commands to run tests
+before_script: redis-cli ping
+
+script:
+  - nosetests
+
+#after_success:
+#  - codecov
+
+services:
+  - redis-server
```

Comment:
I think this should be changed to setup postgres, not sure if this is why builds are failing. also would double check that the python version is the version specified elsewhere for the project